gymnasium/spaces/tuple.py

```diff
@@ -87,8 +86,8 @@ def seed(
         return seeds
 
     def sample(
-        self, mask: Optional[TypingTuple[Optional[np.ndarray], ...]] = None
-    ) -> tuple:
+        self, mask: TypingTuple[Any | None, ...] | None = None
+    ) -> TypingTuple[Any, ...]:
```

Comment:
`tuple` gives the same information but is simpler

Reply:
I find `Tuple[Any, ...]` is more obvious to the users what the type is compared to `tuple` which I agree is simpler